btr-web/btr-main-app/pages/staffRequestView.vue

```diff
@@ -0,0 +1,347 @@
+      <div data-cy="staff-request-header">
+        <h2 class="text-2xl font-bold mb-2.5">
+          {{ $t('pageHeadings.siManagementView') }}
+        </h2>
+      </div>
+
+      <div class="flex justify-between items-center mb-4">
+        <p data-cy="staff-si-dash-text" />
+      </div>
+
+      <BcrosSection
+        data-cy="siRequestBizInfoSection"
+        :section-title="$t('staffSiView.headers.bizInfo')"
+        section-title-icon="i-mdi-domain"
+        section-icon-color="text-bcGovColor-footer"
+        rounded-bot
+        rounded-top
+        :padded-top="false"
+        :padded-x="false"
+        :padded-y="false"
+        class="mb-10"
+      >
+        <template #section-title>
+          <BcrosStaffSectionHeader
+            datacy="bizInfo"
+            :title="$t('staffSiView.headers.bizInfo')"
+            icon="i-mdi-domain"
+          />
+        </template>
+
+        <template #default>
+          <div class="px-8 py-4">
+            <BcrosInfoLine
+              datacy="bizName"
+              :label="$t('staffSiView.labels.bizName')"
+              :value="activeRequest?.business?.legalName"
+            />
+            <BcrosInfoLine
+              datacy="corpNo"
+              :label="$t('staffSiView.labels.corpNo')"
+              :value="activeRequest?.businessIdentifier"
+            />
+            <BcrosInfoLine
+              datacy="filingDate"
+              :label="$t('staffSiView.labels.siFilingDate')"
+              :value="formatDate(activeRequest?.createdAt)"
+            />
+            <BcrosInfoLine
+              datacy="days"
+              :label="$t('staffSiTable.days')"
+              :value="numDaysBetweenNow(activeRequest?.createdAt)"
+            />
+          </div>
+        </template>
+      </BcrosSection>
+
+      <BcrosSection
+        data-cy="siRequestSISection"
+        :section-title="$t('staffSiView.headers.si')"
+        section-title-icon="i-mdi-account-cog"
+        section-icon-color="text-bcGovColor-footer"
+        rounded-bot
+        rounded-top
+        :padded-top="false"
+        :padded-x="false"
+        :padded-y="false"
+        class="mb-10"
+      >
+        <template #section-title>
+          <BcrosStaffSectionHeader
+            :title="$t('staffSiView.headers.si')"
+            icon="i-mdi-account-cog"
+          />
+        </template>
+
+        <template #default>
+          <div class="px-8 py-4">
+            <BcrosInfoLine
+              datacy="fullName"
+              :label="$t('staffSiView.labels.fullName')"
+              :value="activeRequest?.fullName"
+            />
+            <BcrosInfoLine
+              datacy="born"
+              :label="$t('staffSiView.labels.born')"
+              :value="formatDate(activeRequest?.birthdate, false)"
+            />
+            <!-- <BcrosInfoLine
+              :label="$t('staffSiView.labels.citizenship')"
+              :value="activeRequest?.birthdate"
+            /> -->
```

Comment:
Can we remove this or are you expecting to uncomment it later ?

Reply:
Both, I can remove it for now, it was in the mockups but I confirmed we don't need it for now as it would require some best match pairing with existing SIs for the business which we aren't doing for now